contracts/Uniswap/UniswapConfig.sol

```diff
@@ -538,39 +612,31 @@ contract UniswapConfig {
         reporterMultiplier22 = get(configs, 22).reporterMultiplier;
         reporterMultiplier23 = get(configs, 23).reporterMultiplier;
         reporterMultiplier24 = get(configs, 24).reporterMultiplier;
+        reporterMultiplier25 = get(configs, 25).reporterMultiplier;
+        reporterMultiplier26 = get(configs, 26).reporterMultiplier;
+        reporterMultiplier27 = get(configs, 27).reporterMultiplier;
+        reporterMultiplier28 = get(configs, 28).reporterMultiplier;
+        reporterMultiplier29 = get(configs, 29).reporterMultiplier;
+        reporterMultiplier30 = get(configs, 30).reporterMultiplier;
+        reporterMultiplier31 = get(configs, 31).reporterMultiplier;
+        reporterMultiplier32 = get(configs, 32).reporterMultiplier;
+        reporterMultiplier33 = get(configs, 33).reporterMultiplier;
+        reporterMultiplier34 = get(configs, 34).reporterMultiplier;
 
-        isUniswapReversed00 = get(configs, 0).isUniswapReversed;
-        isUniswapReversed01 = get(configs, 1).isUniswapReversed;
-        isUniswapReversed02 = get(configs, 2).isUniswapReversed;
-        isUniswapReversed03 = get(configs, 3).isUniswapReversed;
-        isUniswapReversed04 = get(configs, 4).isUniswapReversed;
-        isUniswapReversed05 = get(configs, 5).isUniswapReversed;
-        isUniswapReversed06 = get(configs, 6).isUniswapReversed;
-        isUniswapReversed07 = get(configs, 7).isUniswapReversed;
-        isUniswapReversed08 = get(configs, 8).isUniswapReversed;
-        isUniswapReversed09 = get(configs, 9).isUniswapReversed;
-        isUniswapReversed10 = get(configs, 10).isUniswapReversed;
-        isUniswapReversed11 = get(configs, 11).isUniswapReversed;
-        isUniswapReversed12 = get(configs, 12).isUniswapReversed;
-        isUniswapReversed13 = get(configs, 13).isUniswapReversed;
-        isUniswapReversed14 = get(configs, 14).isUniswapReversed;
-        isUniswapReversed15 = get(configs, 15).isUniswapReversed;
-        isUniswapReversed16 = get(configs, 16).isUniswapReversed;
-        isUniswapReversed17 = get(configs, 17).isUniswapReversed;
-        isUniswapReversed18 = get(configs, 18).isUniswapReversed;
-        isUniswapReversed19 = get(configs, 19).isUniswapReversed;
-        isUniswapReversed20 = get(configs, 20).isUniswapReversed;
-        isUniswapReversed21 = get(configs, 21).isUniswapReversed;
-        isUniswapReversed22 = get(configs, 22).isUniswapReversed;
-        isUniswapReversed23 = get(configs, 23).isUniswapReversed;
-        isUniswapReversed24 = get(configs, 24).isUniswapReversed;
+        TokenConfig memory config;
+        uint64 isUniswapReversed_ = 0;
```

Comment:
Out of interest, what's the gas difference of doing this instead of an immutable bool per config?

Reply:
Unfortunately I wasn't paying much attention to exactly how much difference it made with the extra bitshifting required to read them, but it doesn't seem to be that much at all. Some of the updated gas values in tests: https://github.com/smartcontractkit/open-oracle/pull/4/files#diff-64e06005b50c2c458f2fff2702e45630fa7e17eedc1787d3179e96a698f5bf10R190